tornado/httputil.py

```diff
@@ -783,7 +783,9 @@ def parse_body_arguments(
             )
             return
         try:
-            uri_arguments = parse_qs_bytes(native_str(body), keep_blank_values=True)
+            uri_arguments = parse_qs_bytes(
```

Comment:
Add a comment like "Use latin1 to pass the raw bytes through for now. The real decoding will happen later in RequestHandler.decode_argument"

Maybe `parse_qs_bytes` should take bytes instead of str and do this conversion internally? The other call to `parse_qs_bytes` (for the query string) need an encode in the other direction, so it doesn't simplify things much, but it does seem conceptually cleaner if `parse_qs_bytes` has a bytes-only interface.